sw/airborne/firmwares/rotorcraft/autopilot_arming_throttle.h

```diff
@@ -117,14 +117,14 @@ static inline void autopilot_arming_check_motors_on(void)
         }
         break;
       case STATE_MOTORS_OFF_READY:
-        autopilot.motors_on = false;
+        autopilot_set_motors_on(false, false);
```

Comment:
I think you made a typo here ? (2 parameters ?)

Reply:
Aha yes, I had that first when I wanted the second one to forcefully bypass the motors on.
This PR is just a simple showcase (not 100% tested and finished) on how I might wanted to implement this feature.